ci/build_container/build_container.sh

```diff
@@ -39,59 +41,59 @@ export CC=gcc-4.9
 export CXX=g++-4.9
 
 # libevent
-wget https://github.com/libevent/libevent/releases/download/release-2.1.8-stable/libevent-2.1.8-stable.tar.gz
+wget ${LIBEVENT_HTTP_ARCHIVE}
 tar xf libevent-2.1.8-stable.tar.gz
-cd libevent-2.1.8-stable
-./configure --prefix=$THIRDPARTY_BUILD --enable-shared=no --disable-libevent-regress --disable-openssl
+cd ${LIBEVENT_PREFIX}
+./configure --prefix=${THIRDPARTY_BUILD} --enable-shared=no --disable-libevent-regress --disable-openssl
 make install
 cd ..
 rm -fr libevent*
 
 # BoringSSL
-git clone https://boringssl.googlesource.com/boringssl
+git clone ${BORINGSSL_REMOTE}
 cd boringssl
-git reset --hard b87c80300647c2c0311c1489a104470e099f1531
-cmake .
-make
-cp -r include/* $THIRDPARTY_BUILD/include
-cp ssl/libssl.a $THIRDPARTY_BUILD/lib
-cp crypto/libcrypto.a $THIRDPARTY_BUILD/lib
+git reset --hard ${BORINGSSL_COMMIT}
```

Comment:
Sorry I don't understand what the issue is here. Why do we need to build boringssl with bazel? Can't we just build it in this file like we are building everything else (using package native build system).